README.md

```diff
@@ -171,20 +171,46 @@ includes:
 - Flask Integration
 - PyMongo Integration
 
+The v0.4 alpha release includes:
+
+- Metrics MinMaxSumCount Aggregator
+- Context API 
+- Full Metrics SDK Pipeline 
+- Metrics STDOUT Exporter
+- Dbapi2 Integration
+- MySQL Integration
+- Psycopg2 Integration
+- Zipkin Exporter
+- Added Examples
+- Additional and Improved Examples
+- Prometheus Metrics Exporter
+
+Thank you to the following individuals for contributing to this release:
+
+* Alex Boten
+* Chris Kleinknecht
+* Christian Neumüller
+* Daniel González
+* Diego Hurtado
+* Golovin Pavel
+* Hector Hernandez
+* Jake Malachowski
+* Joshua H Lang
+* Leighton Chen
+* Mauricio Vásquez
+* Yusuke Tsutsumi
+
 See the [project
 milestones](https://github.com/open-telemetry/opentelemetry-python/milestones)
 for details on upcoming releases. The dates and features described here are
 estimates, and subject to change.
 
 Future releases targets include:
 
-| Component                           | Version    | Target Date       |
-| ----------------------------------- | ---------- | ----------------- |
-| Zipkin Trace Exporter               | Alpha v0.4 | February 21 2020  |
-| W3C Correlation Context Propagation | Alpha v0.4 | February 21 2020  |
-| Support for Tags/Baggage            | Alpha v0.4 | February 21 2020  |
-| Metrics Aggregation                 | Alpha v0.4 | February 21 2020  |
-| gRPC Integrations                   | Alpha v0.4 | February 21 2020  |
-| Prometheus Metrics Exporter         | Alpha v0.4 | February 21 2020  |
-| OpenCensus Bridge                   | Alpha v0.4 | February 21 2020  |
-| Metrics SDK (Complete)              | Alpha v0.4 | February 21 2020  |
+| Component                           | Version    | Target Date  |
+| ----------------------------------- | ---------- | ----         |
```

Comment:
OT Collector exporters are not mentioned, these are being part of Beta release checklist so it would be good to add it here